src/include/function/udf_function.h

```diff
@@ -226,29 +226,29 @@ struct UDF {
         validateType<TR>(returnType);
         scalar_exec_func scalarExecFunc = getScalarExecFunc<TR, Args...>(udfFunc, parameterTypes);
         definitions.push_back(std::make_unique<function::ScalarFunction>(
-            name, std::move(parameterTypes), returnType, std::move(scalarExecFunc)));
+            std::string(name), std::move(parameterTypes), returnType, std::move(scalarExecFunc)));
```

Comment:
It doesn't make sense to use `std::string_view` in the API when we are going to construct a string regardless. In this case, we should take a `std::string` by value and let the user decide how to construct it.

Reply:
That's true. Though it brings up the question of do we want to prefer `string_view` anyway for #2521 (noting again that it's still not working in that context for other reasons), or should we take the more performant route to taking strings by value and moving them when we want ownership? Maybe we should go with the latter for the moment, but eventually move towards the former when and if the rest of what's required for #2521 is implemented?

Reply:
Personally, I think we should just distribute a debug version of our windows DLL. Other approaches hurt everyone across the board.